### DISCUSSION TITLE
T0 creación de entidades

### DISCUSSION THREAD

**Discussion by vale1206** [IIC2133-PUC/2024-2#9] [Tarea 0] [answered]
Hola, vi que en el archivo main.c hay una sección de construcción de entidades y me entro la duda si es necesario modularizar el codigo o basta solo con trabajar en el main.

**Comment by sebacarrascop** [comment, marked as answer]
Hola!
Puedes hacer lo que te sea más cómodo. En general es buena practica modularizar.
Recuerda que pueden modificar como quieran el código base

**Reply by jorgee02** [reply]
Hola! En base a la observación que hacen, qué tanto se puede modificar el código base? Me refiero, en particular, a si es que es posible incluso no usar las funciones estáticas que aparecen definidas, cambiar los prints e incluirlos en las funciones, etc. 
Gracias

**Reply by tomastrivino** [reply]
@jorgee02 Puedes hacer lo que estimes conveniente, siempre y cuando cumplas con lo que se pide en la tarea. El código base sólo son hints; puedes cambiar lo que desees.

**Comment by Mikson16** [comment]
¿Si realizo un módulo para las entidades y luego agrego el contenido de ese módulo (estructuras) al main, también tengo que modificar el Makefile? ¿Para que considere ese nuevo módulo?
